tools/Customization/DevHome.FileExplorerSourceControlIntegration/Services/RepositoryTracking.cs

```diff
@@ -0,0 +1,180 @@
+﻿// Copyright (c) Microsoft Corporation.

```

Comment:
NIT for the file: Make CLSID the second argument.  That is what it is.  All other methods need either
1. nothing
2. RootPath
3. RootPath and CLSID.

This keeps the arguments in a consistent order.